nncf/quantization/algorithms/bias_correction/onnx_backend.py

```diff
@@ -88,45 +99,35 @@ def mean_statistic_collector(reduction_shape: ReductionShape,
     def batch_statistic_collector(num_samples: int = None) -> ONNXMeanStatisticCollector:
         return ONNXBatchStatisticCollector(num_samples)
 
-    @staticmethod
-    def get_tensor_names(node: NNCFNode):
-        return node.layer_attributes.input_tensor_names, \
-            node.layer_attributes.output_tensor_names
-
     @staticmethod
     def process_model_output(raw_data: Dict, output_name: str) -> ONNXNNCFTensor:
         return ONNXNNCFTensor(raw_data[output_name])
 
     @staticmethod
-    def get_node_through_quantizer(node: NNCFNode, nncf_graph: NNCFGraph) -> NNCFNode:
-        activation_input_port = 0
-        quantizer_type = ONNX_OPERATION_METATYPES.get_operator_metatype_by_op_name('QuantizeLinear')
-        dequantizer_type = ONNX_OPERATION_METATYPES.get_operator_metatype_by_op_name('DequantizeLinear')
-        skip_types = dequantizer_type.op_names + quantizer_type.op_names
-        previous_node = nncf_graph.get_previous_nodes(node)[activation_input_port]
-        while previous_node.node_type in skip_types:
-            previous_node = nncf_graph.get_previous_nodes(previous_node)[activation_input_port]
-        return previous_node
-
-    @staticmethod
-    def get_activation_port_ids_for_bias_node(model: onnx.ModelProto, node: NNCFNode) -> Tuple[int, int]:
+    def get_activation_port_ids_for_bias_node(node: NNCFNode) -> Tuple[int, int]:
         return 0, 0
 
     @staticmethod
-    def get_bias_value(node: NNCFNode, model: onnx.ModelProto) -> np.ndarray:
-        get_bias_value(node, model)
+    def get_bias_value(node: NNCFNode, model: onnx.ModelProto, nncf_graph: NNCFGraph) -> np.ndarray:
+        return get_bias_value(node, model)
 
     @staticmethod
-    def get_bias_port_id(model: onnx.ModelProto, node: NNCFNode) -> int:
+    def get_bias_port_id(node: NNCFNode, model: onnx.ModelProto) -> int:
         onnx_graph = ONNXGraph(model)
         onnx_node = onnx_graph.get_node_by_name(node.node_name)
         return onnx_graph.get_bias_tensor_port_id(onnx_node)
 
     @staticmethod
-    def get_output_names(model: onnx.ModelProto, node_name: str) -> List[str]:
+    def get_input_name(model: onnx.ModelProto, node_name: str) -> str:
+        onnx_graph = ONNXGraph(model)
+        node = onnx_graph.get_node_by_name(node_name)
+        return node.input[0]
+
+    @staticmethod
+    def get_output_name(model: onnx.ModelProto, node_name: str) -> List[str]:
         onnx_graph = ONNXGraph(model)
         node = onnx_graph.get_node_by_name(node_name)
-        return node.output
+        return node.output[0]
 
     @staticmethod
     def extract_model(model: onnx.ModelProto,
```

Comment:
Could you please move this logic to `ONNXModelTransformer` and update FBC algorithm (`get_input_output_names()` method should be removed from backend)?

Reply:
Could you please check maybe it is possible to remove `ONNXExtendedLayerAttributes` class?

Reply:
`ONNXExtendedLayerAttributes` can't be removed due to their's usage in the `node_utils.py`.